### ISSUE TITLE
NPC List

### ISSUE THREAD

**Issue by DraksterAU** [StargateMC/StargateMCPublic#100]
Passive List:

- Civilian (Basic Interactions, space-filler)
- Scientist/Researcher (Unique interactions, space-filler)

Agressive List:
- Lone Wolf /Explorer (eg: Wraith Hunters fall under this, as they come from various races)
- Soldier (Generic Combat NPC, will reward reputation and some basic-ish rewards on death)
- Advanced Soldier (Harder to kill soloable "Boss" level NPC... drops some cool stuff rarely)
- VIP (This would be unique NPCs like Rodney, Sam, Oneill etc)
- Faction Leader (Wraith Queen, Goauld, Elizabeth, Hank Landry, etc)

Basically we need 7 NPC "templates" that are generalised and can be reskinned, renamed, have their interactions changed and their loot tables adjusted slighty before being assigned to various factions. Allows for everything to remain balanced.

@mainesi @Maxstripe @mbacookie Any problems? @XredstonemodsX You'd be great at this...
